acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/utils/HashicorpVault.java

```diff
@@ -16,19 +16,37 @@
 
 import com.github.dockerjava.api.DockerClient;
 
-public class HashicorpNode {
+public class HashicorpVault {
 
-  public HashicorpVaultDocker start(final DockerClient docker) {
+  private final HashicorpVaultDocker dockerContainer;
+
+  public HashicorpVault(final HashicorpVaultDocker dockerContainer) {
+    this.dockerContainer = dockerContainer;
+  }
+
+  public static HashicorpVault start(final DockerClient docker) {
```

Comment:
nit: not sure if this name is quite right now as it both creates and starts the vault.

Reply:
changed to "createVault" - the concept of "Starting" a vault is odd, right?